src/plugins/kdb/ldap/libkdb_ldap/ldap_misc.c

```diff
@@ -1330,6 +1330,43 @@ free_princ_ent_contents(osa_princ_ent_t princ_ent)
     princ_ent->old_key_len = 0;
 }
 
+/* Get any auth indicator values from LDAP and update the "require_auth"
+ * string */
+static krb5_error_code
+get_ldap_auth_ind(krb5_context context, LDAP *ld, LDAPMessage *ldap_ent,
+                  krb5_db_entry *entry)
+{
+    krb5_error_code ret;
+    int i;
+    char **auth_inds = NULL;
+    struct k5buf buf = EMPTY_K5BUF;
+
+    auth_inds = ldap_get_values(ld, ldap_ent, "krbPrincipalAuthInd");
+    if (auth_inds == NULL)
+        return 0;
+
+    k5_buf_init_dynamic(&buf);
+
+    /* space seperated list of indicators */
```

Comment:
Comments should usually be complete, punctuated sentences; in this case, I suggest "Make a space-separated list of indicators."  There are other comments in this commit which are complete sentences but lack punctuation; I won't comment on all of them but please fix them as well.